tiny-event-sourcing-lib/src/main/kotlin/ru/quipy/domain/EventSourcingDomain.kt

```diff
@@ -37,6 +39,7 @@ data class EventRecord(
     val aggregateVersion: Long,
     val eventTitle: String,
     val payload: String,
+    var sagaContext: SagaContext? = SagaContext(),
```

Comment:
both in event and context? why? And if required -> comment, maybe containing the code example. Bacause it is not obvious

Reply:
In event - to get information(`SagaContext`) in the subscriber. In `EventRecord` - to get `SagaContext` in `SagaEventStream`. Otherwise it would have to be mapped into the event somehow.
We can try to make it so that `SagaContext` is stored only in the event record in the database, and when mapping, it is inserted into the event.